shoop/core/models/_contacts.py

```diff
@@ -234,12 +239,18 @@ class Meta:
         verbose_name_plural = _('persons')
 
     def save(self, *args, **kwargs):
+        if self.first_name or self.last_name:
+            self.name = (" ".join([self.first_name, self.last_name])).strip()
```

Comment:
Hmm.. now we can't have full names like 'John Fitzgerald "Jack" Kennedy' anymore. Should we have person.middle_names too? This would then be
```python
names = [self.first_name, self.middle_names, self.last_name]
if any(names):
    self.name = " ".join(x for x in names if x)
```
What do you think?